woocommerce/functions.php

```diff
@@ -47,7 +47,7 @@ function siteorigin_north_woocommerce_enqueue_styles( $styles ){
 		);
 	}
 
-	if( function_exists('is_woocommerce') && is_woocommerce() && siteorigin_setting( 'responsive_disabled' ) ) {
+	if( function_exists('is_woocommerce') && siteorigin_setting( 'responsive_disabled' ) ) {
```

Comment:
You could probably also remove this `function_exists('is_woocommerce')` part too.